### ISSUE TITLE
Framed monster block background image doesn't appear in the PDF

### ISSUE THREAD

**Issue by RKuerten** [naturalcrit/homebrewery#3275]
### Renderer

v3

### Browser

Chrome

### Operating System

Windows

### What happened?

When generating a PDF, the .monster.frame is always generated with the background color, ignoring the background image.

I tested with different background images, with different background-blend-modes (mainly normal, lighten, darken, and multiply), the result is always the same (no background image for the monster block in the PDF).

![Print difference](https://github.com/naturalcrit/homebrewery/assets/30464993/b15b4bbf-9af4-4e6a-81b7-40802f476c82)
Browser vs Generated PDF

Browsers tested: 
Brave V 1.62.153 w/ Chromium 121.0.6167.85 64-bits
Chrome V 121.0.6167.86 64-bits

**Comment by Gazook89** [comment]
I just test with a new brew, featuring only a framed stat block, with Chrome on Mac (viewed in Preview), and had no issue.  Hrm.

Are you using the default PDF printer, or something else (CutePDF etc)?  

Is this true with a brand new brew, with no custom css?  Or, can you provide the Share link to your brew that isn't working?

**Comment by RKuerten** [comment]
I can't share the brew publicly because it is a product yet to be released on the DMsGuild. I'm using the default printer (on Windows, btw). 

I've tested with a brand new brew, and it works. It also works by copying a single page of the brew with a stat block with the custom css. The brew is not that big (30 pages), but there's basically an image in every page along side the stat blocks. Could be some memory issue.  

Once I add the pages (cover page & toc + intro page) that should be before the first page with a stat block, the background disappears. If delete both pages, only the first stat block has the background, the others default to the background color.

**Comment by RKuerten** [comment]
I've circumvented the problem by adding the background texture as a before element in the .monster.frame blocks. The code for it looks like this (for .wide blocks, it is equal except for the width, which is 102.1%). 

```
.page .monster.frame::before  {
  background-color     : #F7F0D4;
  background-image     : url(https://i.imgur.com/owy8gdY.jpg);
  background-blend-mode: normal;
  background-size      : 100% 100%;
  background-repeat    : no-repeat;
  background-position  : top center;
  content              : "";
  position             : absolute;
  width                : 104.1%;
  height               : 100%;
  top                  : 0;
  left                 : -7px;
  z-index              : -3;
}
```

I'm closing this issue.

**Comment by 5e-Cleric** [comment]
New brew just created:
<img width="1256" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/05d44108-e6c3-4df4-9045-409e7213bafd">
print view:
<img width="1280" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/065aebed-77ea-4dce-b015-b0427fc4eba7">
PDF after download:
<img width="639" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/57c84848-4df1-4520-bfc8-0df83fc268af">

**Comment by RKuerten** [comment]
Here's a brew with the same problem, no CSS changes from the base v3:

I had a 2-page lore doc for my campaign (avernus) written in Portuguese, added two pages with the same generated stat block in each: no background when generating the PDF. I 

However, I discovered that if I scroll down to the stat block in the print page, the background loads for that stat block (and that stat block only). Additional stat blocks only have the background color.

Here's the share link: https://homebrewery.naturalcrit.com/share/MbQOA3BqiTUz

Tested in both browsers below, on Windows 10:
Brave V 1.62.153 w/ Chromium 121.0.6167.85 64-bits
Chrome V 121.0.6167.86 64-bits

**Comment by calculuschild** [comment]
Looks like we have `background-attachment: fixed` which is causing the background to scroll out of view on the print page. I thought we had fixed this earlier but I guess not. Should be an easy change.

@RKuerten In the mean time you can add this code to your Style tab:

.page .monster.frame {
  background-attachment: unset;
}

**Comment by 5e-Cleric** [comment]
so the bug can't happen in a one page document, good to know.

**Comment by calculuschild** [comment]
For #3297 , the fix should be to just remove the `background-attachment` property entirely, instead of setting it to `unset`.

**Comment by RKuerten** [comment]
Created a new pull request to fix that.

**Comment by RKuerten** [comment]
Fixed with the merge of #3330
